src/index.js

```diff
@@ -5,6 +5,16 @@ const crypto = require('./crypto')
 
 module.exports = Multihashing
 
+/**
+ * Hash the given `buf` using the algorithm specified
+ * by `func`.
+ *
+ * @param {Buffer} buf - The value to hash.
+ * @param {number|string} func - The algorithm to use.
```

Comment:
I'm not sure I understand, is your issue the name `func`, because the definition I wrote is `number` or `string`.